app/schemas/params.js

```diff
@@ -0,0 +1,7 @@
+exports.isId = {
+  id: {
+    in: ['params'],
+    errorMessage: 'Please enter a valid id',
+    isInt: true
```

Comment:
If you want, you can add a sanitizer here to parse id into number.
`toInt: true`